code/clue.js

```diff
@@ -1,59 +1,350 @@
-}
+  image: "assets/green.png",
+  occupation: "Entrepreneur",
+  favouriteWeapon: "knife",
+};
+
+const professorPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "purple",
+  description: "He has a lot of medicin knowledge",
+  age: 40,
+  image: "assets/plum.png",
+  occupation: "professor",
+  favouriteWeapon: "rope",
+};
+
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlet",
+  color: "red",
+  description: "She is creative",
+  age: 35,
+  image: "assets/scarlet.png",
+  occupation: "architect",
+  favouriteWeapon: "candlestick",
+};
+
+const mrsPeackok = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "blue",
+  description: "She is free-minded",
+  age: 55,
+  image: "assets/peacock.png",
+  occupation: "actress",
+  favouriteWeapon: "dumbbell",
+};
+
+const colonelMustard = {
+  firstName: "Jack",
+  lastName: "Mustard",
+  color: "yellow",
+  description: "He is strong",
+  age: 34,
+  image: "assets/mustard.png",
+  occupation: "butcher",
+  favouriteWeapon: "poison",
+};
+
+const mrsWhite = {
+  firstName: "Ana",
+  lastName: "White",
+  color: "white",
+  description: "She is very weak",
+  age: 28,
+  image: "assets/white.png",
+  occupation: "painter",
+  favouriteWeapon: "axe",
+};
 
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
 
 const rope = {
-  name: 'rope',
-  weight: 10
-}
+  name: "rope",
+  weight: 10,
+  id: "rope",
+};
+
+const knife = {
+  name: "knife",
+  weight: 5,
+  id: "knife",
+};
+
+const candlestick = {
+  name: "candlestick",
+  weight: 7,
+  id: "candlestick",
+};
+
+const dumbbell = {
+  name: "dumbbell",
+  weight: 30,
+  id: "dumbbell",
+};
+
+const poison = {
+  name: "poison",
+  weight: 2,
+  id: "poison",
+};
+
+const axe = {
+  name: "axe",
+  weight: 12,
+  id: "axe",
+};
+
+const bat = {
+  name: "bat",
+  weight: 13,
+  id: "bat",
+};
+
+const trophy = {
+  name: "trophy",
+  weight: 20,
+  id: "trophy",
+};
+
+const pistol = {
+  name: "pistol",
+  weight: 90,
+  id: "pistol",
+};
 
 // THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
 
+console.log(colonelMustard.color);
+console.log(pistol.weight);
+
 // NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
-]
+  professorPlum,
+  missScarlet,
+  mrsPeackok,
+  colonelMustard,
+  mrsWhite,
+];
+
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+];
+
+const rooms = [
+  "Dining Room",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "Billiard Room",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "Living Room",
+  "Observatory",
+  "Theater",
+  "Guest House",
+  "Patio",
+];
 
-const weapons = []
+console.log(suspects);
+console.log(suspects[3].occupation);
+console.log(weapons);
+console.log(rooms.indexOf("Spa"));
 
-const rooms = []
+const weaponRooms = {
```

Comment:
Such a good idea to solve the problem this way, to make certain weapons more likely in certain rooms. I like that you can be so flexible, having e.g. only one room for the knife, but many for the candlestick!